### PR TITLE
Add new GetMempoolEntry RPC method 2

### PR DESCRIPTION
**Pull request by nopara73**
Replacement of: https://github.com/MetacoSA/NBitcoin/pull/611

I added XML comments and removed an obsolated entry.

### CODE REVIEW

File: NBitcoin.TestFramework/NodeBuilder.cs [hunk 1]
```diff
@@ -573,7 +573,7 @@ public void Broadcast(params Transaction[] transactions)
 			{
 				batch.SendRawTransactionAsync(tx);
 			}
-			rpc.SendBatch();
+			batch.SendBatch();
```
Comment (NicolasDorier) [inline NBitcoin.TestFramework/NodeBuilder.cs]: wait how it could even work before?

**Comment by NicolasDorier** [comment]
I would `throwIfNotFound` to `false` by default

File: NBitcoin.TestFramework/NodeBuilder.cs (hunk 1 above)
Comment (lontivero) [reply NBitcoin.TestFramework/NodeBuilder.cs]: It didn't work before, in fact it threw an exception saying "use PrepareBatch".

**Comment by NicolasDorier** [comment]
shit indeed, keep consistent.

**Pull request merged**